Dockerfile.web

```diff
@@ -0,0 +1,25 @@
+# Install dependencies
+
+FROM composer AS dependencies
+
+RUN mkdir -p /var/www/html
+WORKDIR /var/www/html
+
+COPY . /var/www/html
+
+RUN composer install
+
+# Serve website
+
+FROM php:apache
+
+RUN mkdir -p /var/www/html
+WORKDIR /var/www/html
+
+COPY --from=dependencies /var/www/html /var/www/html
+
+RUN docker-php-ext-install pdo_mysql mysqli
```

Comment:
I think it's missing the redis client for predis

Reply:
Predis is installed with Composer and does not need any additional extensions. But do you mean `ext-phpiredis` which is suggested extension by Predis and it "Allows faster serialization and deserialization of the Redis protocol".

Reply:
I was not aware that predis had a solely PHP option so I compiled the ext-php7.3-redis library. Thanks for letting me know there is a raw PHP version.

Reply:
Yes, IndieLogin is using Predis so there is no need for extensions.